tools/data_deletion/README.md

```diff
@@ -0,0 +1 @@
+This Directory contains tools that delete entries from the database.
```

Comment:
here im thinking the delete_candidate and delete_timepoint but I will wait for all open PRs to be merged first

@ZainVirani :1st_place_medal:

Reply:
why not something more generic like "dangerous_scripts"?

Reply:
euhm... dangerous scripts seems too generic !! every script making modifications to the DB is "dangerous".

I think i'm gonna go with @me-evans suggestion of subdividing the data_integrity one into fixers, deleters, and  checkers (which dont modify anything)

Reply:
I think there's already too many categories. It's going to get hard to figure out where scripts are if it's divided too far and you need to guess which categories or subcategories scripts that you know you're looking for are in. We only need a taxonomy that handles 34 scripts (currently)..

Reply:
okai, I think my final offer is 4 directories

`data_integrity`, `deprecated`, `single_use`, `couchdb`

Reply:
So you just dropped this category? Those seem fine, but I'm not opposed to separating out the more dangerous ones that do deletions into another category so that users think twice before using them. I don't even really have anything against "data_deletion", I just think that it would be better to use a more generic category so that we don't need to keep adding new categories for other dangerous scripts.

(Also: "couchdb" -> "data_exporters" (as per above thread))